playbooks/byo/openshift-checks/health.yml

```diff
@@ -1,3 +1,10 @@
 ---
 - include: ../openshift-cluster/initialize_groups.yml
+  tags:
+  - always
+
+- include: ../../common/openshift-cluster/std_include.yml
```

Comment:
Oh, hm; right. So, the question is would it be better to just put in `common/openshift-checks/health.yml`

    - include: ../openshift-cluster/std_include.yml

This pattern would help in the adhoc playbook.